apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.TargetedSessionPool.cs

```diff
@@ -36,7 +36,7 @@ internal sealed class TargetedSessionPool : SessionPoolBase
             // Read-only state
             private readonly object _lock = new object();
             private readonly DatabaseName _databaseName;
-            private readonly CreateSessionRequest _createSessionRequest;
+            private readonly BatchCreateSessionsRequest _batchCreateSessionRequest;
```

Comment:
It may be worth adding a comment to note that this should be cloned each time it's used.